### PR TITLE
add pt distributions for filtered/unfiltered tracks with positive/neg…

### PR DESCRIPTION
**Pull request by Mingyu3360715**

### CODE REVIEW

File: DPG/Tasks/AOTTrack/qaEventTrack.cxx [hunk 1]
```diff
@@ -1315,6 +1319,12 @@ void qaEventTrack::fillRecoHistogramsGroupedTracks(const C& collision, const T&
   int nPvContrWithTOF = 0;
   int nPvContrWithTRD = 0;
   for (const auto& trackUnfiltered : tracksUnfiltered) {
+    // fill unfiltered track pt
+    if (trackUnfiltered.sign() > 0) {
+      histos.fill(HIST("Tracks/Kine/ptUnfilteredPositive"), trackUnfiltered.pt());
+    } else if (trackUnfiltered.sign() < 0) {
```
Comment (mfaggin) [inline DPG/Tasks/AOTTrack/qaEventTrack.cxx:1325]:
```suggestion
    } else {
```

File: DPG/Tasks/AOTTrack/qaEventTrack.cxx [hunk 2]
```diff
@@ -1356,6 +1366,11 @@ void qaEventTrack::fillRecoHistogramsGroupedTracks(const C& collision, const T&
     }
     // fill kinematic variables
     histos.fill(HIST("Tracks/Kine/pt"), track.pt());
+    if (track.sign() > 0) {
+      histos.fill(HIST("Tracks/Kine/ptFilteredPositive"), track.pt());
+    } else if (track.sign() < 0) {
```
Comment (mfaggin) [inline DPG/Tasks/AOTTrack/qaEventTrack.cxx:1371]:
```suggestion
    } else {
```

**Review by mfaggin — CHANGES_REQUESTED** [review]

**Review by mfaggin — APPROVED** [review]

**Comment by mfaggin** [comment]
hi @TimoWilken it seems that we have the issue on GEANT compilation. Any news on it?
Thanks a lot.

**Comment by TimoWilken** [comment]
Hi @mfaggin, looks like the builder hasn't picked this PR up since before the fix was merged. I've manually restarted the check on this and other O2Physics PRs to reduce the waiting time!

**Comment by mfaggin** [comment]
thank you @TimoWilken !

**Pull request merged**